app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -198,7 +197,11 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
             if (p1 != null) {
                 if (p1.action.equals(WifiManager.SCAN_RESULTS_AVAILABLE_ACTION)) {
                     val wifiList = mainWifi.getScanResults()
-                    deviceConnectPresenter.availableDevices(wifiList)
+                    Timber.d("Check " + checkDevice)
```

Comment:
"Check %s", checkDevice

Reply:
I have to add lint suppressing for that so I left it like this only